### DISCUSSION TITLE
How long does it run? It is running for a week now.

### DISCUSSION THREAD

**Discussion by shailabhr** [gephi/gephi#2328] [Q&A]
## Current Behavior
Hi, I am running the Gephi on my Ubuntu system (60 Gb ram, Xeon w-2275 processor, two RTX-5000 GPU card). My file has 186346 nodes and 221129 edges. I am running the ForceAtlas2 module. It is running for a week now. How long do you expect it to run? When I checked the GPU settings, only one GPU is running. Is it a GPU intensive program? Last time it was running for 5 days. I stopped it thinking that there must be something wrong. I restarted it and now it is running for a week. 
## Possible Solution
Kindly help. 
Thank you. 

## Steps to Reproduce

1.
2.
3.
4.

## Your Environment
Ubuntu 18

* Version used: Gephi 0.9.2
* Java version: openjdk 11.0.10
* Operating System: Ubuntu 18

**Comment by michael-imbeault** [comment]
ForceAtlas doesn't end - you need to end it manually when you're satisfied with the result, it will reach some sort of equilibrium after a while.

**Comment by shailabhr** [comment]
Oh, I did not know about it. Thank you. So, should I just stop it and go further? When I stopped it last time, I tried zooming in and out, I could hardly see anything... For my case (186346 nodes and 221129 edges), how long do you think I should run it? Should I stop and save it separately? I would appreciate it if you can suggest to me an instruction document maybe. Thank you.
